esphome/components/json/json_util.cpp

```diff
@@ -54,10 +54,10 @@ void parse_json(const std::string &data, const json_parse_t &f) {
   const size_t free_heap = heap_caps_get_largest_free_block(MALLOC_CAP_INTERNAL);
```

Comment:
Change this too to be consistent?
```suggestion
  const size_t free_heap = heap_caps_get_largest_free_block(MALLOC_CAP_8BIT);
```